### PR TITLE
Solving rubocop problems

### PR DESCRIPTION
**Pull request by gabrielrumiranda**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -8,32 +10,33 @@ class ApplicationController < ActionController::Base
...
-      return
     end
     @user = current_user
   end
 
+  def login
+    session[:return_to] = url_except_param(request.original_url, :log_in_with)
+    logger.debug "After login will return to #{session[:return_to]}"
+    redirect_to root_path(log_in_with: params[:log_in_with])
+  end
```
Comment (baob) [inline app/controllers/application_controller.rb]: Is this new method an action in it's own right ? If not, I think it should be a private method.

(Did rubocop do this refactoring ? It's the 2nd time I've seen it today on a 'rubocop fixes' branch  :)

**Comment by baob** [comment]
Can we really be confident that rubocop didn't break anything here, given that we have no tests. (And given that it's not unknown for rubocop to break things even with tests in place).

Maybe I'm being too cautious :-/

**Comment by gabrielrumiranda** [comment]
Some functions I refactory because is so big and make more than one action, buSome functions I refactory because is so big and make more than one action, but now I go make a PR where only solve problems automatic by rubocop

**Comment by gabrielrumiranda** [comment]
#60

**Comment by aspiers** [comment]
I agree with @baob that some real caution is needed here to due to lack of tests.  I have explicitly detailed the way I want to address this in #57 so please read that carefully.

**Comment by aspiers** [comment]
Handling this a different way.

**Pull request closed without merging**